deblur/workflow.py

```diff
@@ -488,16 +548,69 @@ def launch_workflow(seqs_fp, working_dir, read_error, mean_error, error_dist,
             f.write(s.to_fasta())
     # Step 6: Chimera removal
     output_no_chimeras_fp = remove_chimeras_denovo_from_seqs(
-        output_deblur_fp, working_dir)
-    # Step 7: Generate BIOM table
-    deblur_clrs, table = generate_biom_table(seqs_fp=output_no_chimeras_fp,
-                                             uc_fp="%s.uc" % output_trim_fp,
-                                             delim=delim)
-    # Step 8: Write BIOM table to file
-    if table.is_empty():
-        raise ValueError(
-            "Attempting to write an empty BIOM table.")
-    biom_fp = join(working_dir, "%s.biom" % basename(seqs_fp))
-    write_biom_table(table, biom_fp)
-
-    return biom_fp
+        output_deblur_fp, working_dir, threads=threads_per_sample)
+    logger.info('finished processing file')
+    return output_no_chimeras_fp
+
+
+def start_log(level=logging.DEBUG, filename=''):
+    """
+    start the logger for the run
+
+    Parameters
+    ----------
+    levele : logging.DEBUG, logging.INFO etc. for the log level.
```

Comment:
Also, it is missing the `optional` keywrod and the default in the description. Also the description should be in the next line